doc/developer_guide/commands.md

```diff
@@ -43,9 +43,17 @@ Script `start-test-release-build` requires environment variable `GH_TOKEN` to co
 
 The following commands can be used to deploy the infrastructure onto a given AWS account:
 * `setup-ci-codebuild`: Deploy the AWS Codebuild cloudformation stack which will run the ci-test.
-* `setup-vm-bucket`: Deploy the AWS Bucket cloudformation stack which will be used to deploy the VM images.
 * `setup-release-codebuild`: Deploy the AWS Codebuild cloudformation stack which will be used for the release-build.
+<!--
+* `setup-vm-bucket`: Deploy the AWS Bucket cloudformation stack which will be used to deploy the VM images.
 * `setup-vm-bucket-waf`: Deploy the AWS Codebuild cloudformation stack which contains the WAF Acl configuration for the Cloudfront distribution of the VM Bucket.
+-->
```

Comment:
Looks like this is obsolete? can be removed?

Reply:
yes, I was coward.
See next push.